Libplanet.Action.Tests/Mocks/MockAccount.cs

```diff
@@ -0,0 +1,121 @@
+using System.Numerics;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Store.Trie;
+using Libplanet.Types.Assets;
+using Libplanet.Types.Tx;
+
+namespace Libplanet.Action.Tests.Mocks
+{
+    public static class MockAccount
```

Comment:
Eh... I suppose this is something akin to `MockAccountState`. However, it isn't clear what this class does is as this neither inherits an interface nor is there a description for it. This makes it very prone to creating "bad tests". 😣

Reply:
Maybe using `Account` not `MockAccount` is better than that. What about you?

Reply:
Modified documentation from `MockAccountState` would be somewhat helpful. 😥

Reply:
https://github.com/planetarium/libplanet/pull/3392/commits/a42078fb2141e9f2a31d74b82b6f806ca89aa013 from `MockAccountState`